carma_messenger_vehicle_plugin/src/msger_veh_wrapper.py

```diff
@@ -0,0 +1,76 @@
+ #   Copyright (C) 2022 LEIDOS.
+ #
+ #   Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ #   use this file except in compliance with the License. You may obtain a copy of
+ #   the License at
+ #
+ #   http://www.apache.org/licenses/LICENSE-2.0
+ #
+ #   Unless required by applicable law or agreed to in writing, software
+ #   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ #   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ #   License for the specific language governing permissions and limitations under
+ #   the License.
+
+import argparse
+import logging
+from sumo_connector import SumoConnector
+from msger_veh_cfg import MsgerVehicleCfg
+
+def setup_logging(level):
+    numeric_level = getattr(logging, level.upper(), None)
+    if not isinstance(numeric_level, int):
+        raise ValueError('Invalid log level: %s' % level)
+    logging.basicConfig(level=numeric_level, format='%(asctime)s - %(levelname)s - %(message)s')
+
+def run(args):
+    """
+    Executes the vehicle simulation management. This function initializes the vehicle configurations,
+    connects to SUMO via TraCI, and manages the simulation steps and vehicle states.
+
+    Parameters:
+    - args: Command line arguments containing settings for Traci connection and configuration path.
+    type: Namespace
+    """
+    setup_logging(args.log_level)
+    try:
+        msger_veh_cfg = MsgerVehicleCfg(args.msger_veh_cfg_path, log_level=args.log_level)
+        sumo_connector = SumoConnector(args.traci_ip, args.traci_port, args.traci_order_num, log_level=args.log_level)
+        msg_veh_ids = msger_veh_cfg.get_veh_ids()
+        with sumo_connector.traci_handler():
+            while True:
+                sumo_connector.tick()
```

Comment:
Does this tick interfere with the tick in the existing bridge script? Are we ticking twice as fast now since it's being called each timestep?

Reply:
The code is creating another SUMO client with an order set to 3, while there are two existing SUMO client in the system (SUMO Ambassador and EVC). This shouldn't cause a conflict because SUMO's tick mechanism waits until all clients send their tick command via TraCI. As clients must send the tick, SUMO will only advance the simulation once all connected clients have ticked, so we're not double ticking.